libexec/install.ps1

```diff
@@ -22,27 +54,45 @@ function Find-Miniconda {
 
 $CondaIsInstalled = Find-Miniconda
 
-If (-not $CondaIsInstalled) {
+# On GH CI runners, conda is always installed
+If ((-not $CondaIsInstalled) -or ($Env:ncCI)){
     Write-Host "Downloading miniconda3"
     Invoke-WebRequest $MinicondaLatestUrl -OutFile $Env:temp\Miniconda3-latest-Windows-x86_64.exe
-    Invoke-Expression "$Env:temp\Miniconda3-latest-Windows-x86_64.exe /RegisterPython=1 /S /InstallationType=JustMe /D=$CondaInstallationDirectory"
-    Write-Host "Installed miniconda into $CondaInstallationDirectory"
+    Write-Debug "Done"
+    Write-Host "Installing miniconda3"
+    Start-Process -WorkingDirectory "$Env:temp" -Wait -FilePath "Miniconda3-latest-Windows-x86_64.exe" -ArgumentList "/InstallationType=JustMe /S /D=`"$CondaInstallationDirectory`""
```

Comment:
The installation directory is not resolved properly on my Windows 11 (22H2). Removing all the (back)ticks around the directory works for me, i.e. like this:
```powershell
    Start-Process -WorkingDirectory "$Env:temp" -Wait -FilePath "Miniconda3-latest-Windows-x86_64.exe" -ArgumentList "/InstallationType=JustMe /S /D=$CondaInstallationDirectory"
```

Reply:
This is fixed in https://github.com/neuro-conda/neuro-conda/pull/32/commits/6da5fdd454cfcd7c55433232155b9b123eaaa12f